src/options/options.html

```diff
@@ -147,7 +140,7 @@ <h2 data-i18n="__MSG_subtitleMisc__">Miscellaneous settings</h2>
 							<a class="learn-more" data-i18n="__MSG_optionLearnMore__" data-i18n-href="__MSG_optionQrQuietZoneDescrLink__" href="https://github.com/rugk/offline-qr-code/wiki/FAQ#what-is-a-quiet-zone">Learn more</a>
 						</div>
 					</li>
-
+					
```

Comment:
The diff on GitHub shows some unused whitespace here. Better remove that.

Did you use any `editorconfig` file/add-on/tool?

Reply:
Should be fixed now. 
I wasn't using any editorconfig tool